src/main/resources/db/changelog/202310040000000-create-anonymous-certificates.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="202310040000000-1" author="jalbrech">
```

Comment:
Few things here:

Let's add checks here to make sure the table doesn't already exist before adding it. We can also stack the index creation into the table creation, since the index can't exist if the table didn't exist yet.

Also, if we're going to be adding a new cert table (pls no), that's another thing that we need to clean up in the cert cleanup job.

Finally, I noticed this is modifying the schema of stuff that's net-new to the feature as a whole. Since this is being built in a feature branch we can safely manipulate the existing changeset when it makes sense to do so, rather than updating the schema in a separate changeset as we do for stuff that hits a shippable branch. That way we can avoid the pain inherit to having several changesets that have to be merged and ordered together later when the feature branch is merged in.

Reply:
Updated